scalafmt.conf

```diff
@@ -0,0 +1,16 @@
+version = "2.3.2"
```

Comment:
Filename should be .scalafmt.conf, AFAIK.

Reply:
Renamed